use/posts.js

```diff
@@ -110,11 +110,14 @@ export const usePosts = () => {
   const postFields = [
     {
       name: 'username',
+      label: t('posts.edit.label.username'),
       admin: true,
     },
     {
       name: 'type',
       component: 'TInputSelect',
+      label: t('posts.edit.label.type'),
```

Comment:
Could you please re-organise it in the following way: `<object>.<property>.<attribute>`,

In this case it would be: 
- object: post
- property: type
- attribute: label (could be also placeholder, title, description, etc.)

```suggestion
      label: t('post.type.label'),
```

Reply:
@larrycoal and this?